src/users/containers/user-details/user-details.component.ts

```diff
@@ -78,14 +78,15 @@ export class UserDetailsComponent implements OnInit, OnDestroy {
         this.userAccessTypes = [];
         if (isFeatureEnabled) {
           const enabledUserAccessTypes: UserAccessType[] = user?.accessTypes?.filter((x: UserAccessType) => x.enabled) ?? [];
-          const orgAccessTypes: OrganisationAccessType[] = organisationAccessTypes.map((x) => x.accessTypes).reduce((acc, x) => acc.concat(x), []);
-          for (const ac of orgAccessTypes) {
-            if (ac.accessMandatory) {
-              this.userAccessTypes.push(ac.description);
-            }
-            const foundUserAc = enabledUserAccessTypes.find((x) => x.accessTypeId === ac.accessTypeId && x.organisationProfileId === ac.organisationProfileId);
-            if (foundUserAc) {
-              this.userAccessTypes.push(ac.description);
+          for (const jurisdiction of organisationAccessTypes){
+            for (const ac of jurisdiction.accessTypes){
+              if (ac.accessMandatory) {
+                this.userAccessTypes.push(`${jurisdiction.jurisdictionName} - ${ac.description}`);
```

Comment:
Does this need a 'continue' to prevent duplicate values being added for mandatory access types? Or is this handled elsewhere?

Reply:
Yeah this will need a continue else mandatory access types will be shown twice